src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToGenerateImplicitNamespaceImports_WebApp.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToGenerateImplicitNamespaceImports_WebApp : SdkTest
+    {
+        private const string _testProjectName = "TestEmptyWebApp";
+        private const string _importFileName = "EmptyWebApp.ImplicitNamespaceImports.cs";
+
+        public GivenThatWeWantToGenerateImplicitNamespaceImports_WebApp(ITestOutputHelper log) : base(log) { }
+
+        [Fact]
+        public void It_generates_imports_and_builds_successfully()
+        {
+            var tfm = "net6.0";
+            var testAsset = _testAssetsManager
```

Comment:
When using `TestProject`, you can set the `ProjectSdk` property to `Microsoft.NET.Sdk.Web` to use the Web SDK.